src/index.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+// ******* wave 2 *******
+const defaultCity = 'Seattle';
+let number = parseInt(document.querySelector('.number').innerHTML);
+
+const state = { number: number };
+const stateChange = (number) => {
+  if (number >= 80) {
+    document.querySelector('.number').style.color = 'red';
+    document.querySelector('.landscape').innerHTML = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (number >= 70) {
+    document.querySelector('.number').style.color = 'orange';
+    document.querySelector('.landscape').innerHTML = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (number >= 60) {
+    document.querySelector('.number').style.color = 'yellow';
+    document.querySelector('.landscape').innerHTML = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (number >= 50) {
+    document.querySelector('.number').style.color = 'green';
+    document.querySelector('.landscape').innerHTML =
+      '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else {
+    document.querySelector('.number').style.color = 'teal';
+    document.querySelector('.landscape').innerHTML =
+      '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
```

Comment:
This works well! Can you think of a way to rewrite this to make it so you don't need to have so many `getElementById`s repeated?